index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+
```

Comment:
### Your branch name
![pr-title (copy)](https://user-images.githubusercontent.com/93039359/185357368-f021e6b5-5f10-4066-a956-634fe34e0669.png)
**[Optional]**
- [ ] Your branch name is not descriptive enough. You are using **add-2-sections**, you can use  **feature/portfolio-finshed-mobile** :+1: